lib/bookmarks.coffee

```diff
@@ -11,7 +11,7 @@ class ReactBookmarks
       'bookmarks:jump-to-previous-bookmark': @jumpToPreviousBookmark
       'bookmarks:clear-bookmarks': @clearBookmarks
 
-    @markerLayer = @editor.addMarkerLayer({maintainHistory: true})
+    @markerLayer = @editor.addMarkerLayer()
```

Comment:
I'm starting to think that conflating `persist` with `maintainHistory` on the layers is a mistake. Bookmarks are the kind of thing we want to save across reloads / restarts, but we don't need to store them in the undo stack.